src/frontend/src/components/ProposedSolutionForm.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import PageBlock from '../layouts/PageBlock';
+import { Button, Col, Form, InputGroup, Row } from 'react-bootstrap';
+import * as yup from 'yup';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+interface ProposedSolution {
+  description: string;
+  budget: number;
```

Comment:
I think this should be ‘budgetImpact’ such that it matches previous variable naming, but if y’all intentionally want to rename it that’s acceptable. The essence at hand is “how much additional budget is needed?”, so ‘budget’ is definitely not really particularly descriptive. Perhaps ‘additionalBudget’ or ‘additionalBudgetNeeded’ would be good backups if ‘budgetImpact’ is not up to your standards. Also renaming in all places would be best. Both front-end and back-end, but @anthonybernardi should weigh in too

Reply:
if the idea of the feature is additionalBudgetNeeded then yeah the variable should say that and also the label should reflect that so users know what it's referring to

Reply:
BudgetImpact I think gets towards it, i think it makes sense for the spec